build.sbt

```diff
@@ -21,7 +21,7 @@ libraryDependencies ++= Seq(
   "edu.arizona.sista" % "processors" % "3.0" classifier "models",
   "org.slf4j" % "slf4j-api" % "1.7.2",
   "org.slf4j" % "slf4j-log4j12" % "1.7.2",
-  "org.scalatest" %% "scalatest" % "1.9.1" % "test",
+  "org.scalatest" %% "scalatest" % "2.2.6" % "test",
```

Comment:
This was so that I could run a single test from the command line, which was very useful for benchmarking, e.g.:
```
sbt/sbt "test-only nodes.images.ImageBenchMarkSuite -- -z Convolution" > `git rev-parse HEAD`.log
```